auth/src/main/java/com/alibaba/nacos/auth/AbstractProtocolAuthService.java

```diff
@@ -41,40 +41,37 @@
     
     protected final AuthConfigs authConfigs;
     
+    protected final Optional<AuthPluginService> authPluginService;
+    
     protected AbstractProtocolAuthService(AuthConfigs authConfigs) {
         this.authConfigs = authConfigs;
+        this.authPluginService = AuthPluginManager.getInstance()
```

Comment:
It will cause change auth plugin only restart server.
Before your change, you can change plugin only change the configuration.

Reply:
You're right, I didn't  consider that AuthConfigs can be change .